src/main/java/nextstep/ladder/service/LadderGame.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.ladder.service;
+
+import nextstep.ladder.domain.Member;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import nextstep.ladder.domain.Ladder;
+import nextstep.ladder.domain.MemberList;
+import nextstep.ladder.view.InputView;
+import nextstep.ladder.view.ResultView;
+
+public class LadderGame {
+    InputView inputView;
+    ResultView resultView;
+
+    public LadderGame(InputView inputView, ResultView resultView){
+        this.inputView = inputView;
+        this.resultView = resultView;
+    }
+
+    public void operator(){
+        MemberList members = new MemberList();
+        members.setMembers(inputView.getMembers());
+        int height = inputView.getLadderHeight();
+        Ladder ladder = new Ladder(members.getMembers().size(), height);
```

Comment:
```suggestion
        Ladder ladder = new Ladder(members.size(), height);
```
디미터의 법칙에 대해 공부해보면 좋을 것 같아요! 😄